docs/sphinx/examples/cpp/quantinuum

```diff
@@ -0,0 +1,35 @@
+// Compile and run with:
+// ```
+// nvq++ --target quantinuum quantinuum.cpp -o out.x && ./out.x
+// ```
+// Assumes a valid set of credentials have been stored. 
+
+#include <cudaq.h>
+
+// Define a simple quantum kernel to execute on Quantinuum.
+struct ghz {
+  // Maximally entangled state between 5 qubits.
+  auto operator()() __qpu__ {
+    cudaq::qreg q(5);
+    h(q[0]);
+    for (int i = 0; i < 5 - 1; i++) {
```

Comment:
why not just put `i < 4` here?

Reply:
I wrote this as being parameterized on the number of qubits at first and missed this 🤦